python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -237,8 +238,19 @@ def from_pretrained(cls,
 
             # load int x-bit
             kwargs["low_cpu_mem_usage"] = True
-            # set default torch_dtype='auto'
-            kwargs["torch_dtype"] = kwargs.get("torch_dtype", 'auto')
+            # set default torch_dtype='auto'.
+            # Note that when load_in_low_bit="fp16", set default torch_dtype=torch.float16
+            if load_in_low_bit == "fp16":
+                if torch_dtype is not None and torch_dtype != torch.float16:
+                    invalidInputError(
+                        False,
+                        f"Please use torch_dtype=torch.float16"
+                        f" when setting load_in_low_bit='fp16'."
+                    )
+                else:
+                    kwargs["torch_dtype"] = torch.float16
+            else:
+                kwargs["torch_dtype"] = kwargs.get("torch_dtype", 'auto')
```

Comment:
`torch_dtype` seems has been pop in line 145.
Should use `torch_dtype` instead of `kwargs.get("torch_dtype")` here ?

Reply:
Sure.